addons/road-generator/road_point.gd

```diff
@@ -286,7 +282,14 @@ func assign_lanes():
 
 		# Now complete the final lane.
 		if running_same_dir > 0:
-			lanes.append(LaneType.SLOW)
+			if running_same_dir == len(flips):
```

Comment:
Might be good to add a code comment that this is handling the edge case of the extra dotted line textures on the shoulder for the one way street, might be hard to understand this in the future otherwise. Something like:
`#  Special texture case for the "inside" lane of a way one road`

Reply:
Done!